.github/workflows/ci.yml

```diff
@@ -403,6 +403,14 @@ jobs:
           java-version: ${{ env.DEFAULT_JDK_VERSION }}
       - name: Update maven settings
         run: mkdir -p ~/.m2 ; cp .github/settings.xml ~/.m2/
+      - name: Prepare the local distribution archives
+        run: mvn clean install -DskipTests -Pdistribution
+      - name: Run Quarkus Tests in Docker
+        run: |
+          mvn clean install -nsu -B -f quarkus/tests/pom.xml -DkcDistribution=docker | misc/log/trimmer.sh
```

Comment:
Shall we keep the format we are using for properties and have `kcDistribution` as `kc.tests.dist` ? Or something similar.

Also, I'm wondering if it makes more sense to run tests in GHA against docker, only. It should be enough to test not only the image but the distribution itself (as we do for the standalone distribution).

Reply:
I'm happy with whatever name you prefer 🙂

Reply:
Lets stay consistent then, i'd say :)